nerves_ricc_okinawa/lib/sensor/const_nerves.ex

```diff
@@ -3,21 +3,22 @@ defmodule Sensor.ConstNerves do
   定数を定義するためのモジュール
   """
 
-  def my_name(), do: "ricc_okinawa"
+  def my_name(), do: "ricc_piot_kikuyuta"
   def url_temp(), do: "http://" <> System.get_env("NERVES_IP") <> "/api/temp"
   def url_humi(), do: "http://" <> System.get_env("NERVES_IP") <> "/api/humi"
 
   def gpio_button(), do: 5
   def gpio_led(), do: 16
   def led_off(), do: 0
 
-  def i2c_bus(), do: i2c_bus_rpi()
-  def i2c_addr(), do: i2c_addr_mcp9808()
+  def i2c_bus(), do: i2c_bus_bbb()
+  def i2c_addr(), do: i2c_addr_ina226()
```

Comment:
@kikuyuta too dangerous!!
Participants will clone this code for the workshop,,,

Reply:
Oh, sorry,,, 
You have already reverted this change at #12 😁